package.json

```diff
@@ -36,7 +63,7 @@
     "test:systemjs": "node integration/systemjs/systemjs-compatibility-spec.js",
     "test:side-effects": "check-side-effects --test integration/side-effects/side-effects.json",
     "test:side-effects:update": "npm run test:side-effects -- --update",
-    "test:import": "ts-node ./integration/import/runner.ts",
+    "test:import": "node --loader ts-node/esm ./integration/import/runner.ts",
```

Comment:
this is incorrect: test:import is SHOULD NOT use any interop but always use native node.js resolution to test proper import resolution. if we use third party, there's no meaning to run this test.

Reply:
This was an effort to replace the existing interop (ts-node bin).
Since the package is now defined as a module the native nodejs resolution won't work with ts-node.
I think I can address this by treating the integration folder as commonjs and then both of your concerns would be addressed